src/tanks/Tank.js

```diff
@@ -106,7 +106,7 @@ class Tank extends GameObject {
 
 	// Casus:
 	interpriterState: InterpriterState;
-	casusCode: CasusBlock;
+	casusCode: ContainerBlock;
```

Comment:
Yeah, all of these Tank changes look substantially fine to me. As long as they pass flow that means we have been doing things consistently and it shouldn't be a problem for us to make decisions based on this being a ContainerBlock.